Starting job at Tue Feb 18 02:58:05 PM CET 2025
Running Filter_by_part.py for subdirectory 22010, part 10
2025-02-18 14:58:10 - INFO - Event filtering process starts...
2025-02-18 14:58:10 - INFO - Initialized PureNeutrinoEventFilter filter for 22010/10
2025-02-18 14:58:25 - INFO - Loaded 7 event specifier files from /lustre/hpc/project/icecube/HE_Nu_Aske_Oct2024/clean_events_dict/22010/22010/reduced
2025-02-18 14:58:28 - INFO - Selected 2 CSV files covering event range [5114, 6740]
2025-02-18 14:59:00 - INFO - Loaded 20940378 valid (RunID, EventID) pairs from CSV files
2025-02-18 14:59:08 - INFO - Extracted 2793217 valid events based on pure neutrino events.
2025-02-18 14:59:15 - INFO - Initialized CompositeEventFilter filter for 22010/10
2025-02-18 14:59:15 - INFO - Initialized (PureNu)
2025-02-18 14:59:15 - INFO - Starting filtering process for 22010/10
2025-02-18 14:59:15 - INFO - Starting filtering process for 22010/10
2025-02-18 15:01:08 - INFO - Recalculated 'offset' column based on 'N_doms' values within each 'shard_no' group.
2025-02-18 15:01:26 - INFO - Filtered truth file saved to: /lustre/hpc/project/icecube/HE_Nu_Aske_Oct2024/PMTfied_filtered/Snowstorm/PureNu/22010/truth_10.parquet
2025-02-18 15:01:26 - ERROR - An error occurred: 'Field "event_no" does not exist in schema'
Traceback (most recent call last):
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/Filter_by_part.py", line 65, in <module>
    main()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/Filter_by_part.py", line 61, in main
    run()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/Filter_by_part.py", line 55, in run
    filter_manager()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/EventFilterManager.py", line 23, in __call__
    self.filter_obj()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/EventFilter.py", line 32, in __call__
    self._write_filtered_shards()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/EventFilter.py", line 71, in _write_filtered_shards
    pmt_event_nos = pmt_table.column("event_no").to_pylist()
  File "pyarrow/table.pxi", line 1779, in pyarrow.lib._Tabular.column
  File "pyarrow/table.pxi", line 1715, in pyarrow.lib._Tabular._ensure_integer_index
KeyError: 'Field "event_no" does not exist in schema'
Job completed at Tue Feb 18 03:01:26 PM CET 2025
